package.json

```diff
@@ -45,7 +45,7 @@
     "immutability-helper": "^2.8.1",
     "jsonschema": "^1.2.4",
     "material-ui-chip-input": "^1.0.0-beta.8",
-    "mathjs": "^5.2.1",
+    "mathjs": "^7.6.0",
```

Comment:
A general question--from the reviewer's perspective, when you update `package*json`, do I need to review them? I believe you update `package.json`, which updates `package-lock.json`, and you use them to build `node_modules/`. So the update is already tested.

Reply:
Correct, there's certainly no need to review `package-lock.json`—it's autogenerated. Reviewing `package.json` is helpful to see which dependencies were updated—it informs other parts of the PR, like how the update to `mathjs` prompted the changes in `views/EditConfig.react.js`.